joomla/cassiopeia
# 4.0 dev relative sizes

Pull Request for Issue # .

### Summary of Changes

This PR will make the Cassiopeia more a11y by replacing sizes set in `px` by relative sizes like `em` and `rem`.
An interesting blog explaining why we have to do this can be written on 24a11y.com => https://www.24a11y.com/2019/pixels-vs-relative-units-in-css-why-its-still-a-big-deal/

### Testing Instructions

- open Joomla 4 test and inspect elements... font-size, breakpoints and site--grid are set in PX
- open browser settings and increase default font-size.
- see result
- open browser settings and set default font-size back to normal
- apply PR and run `npm run build:css`
- open Joomla 4 test and inspect elements... font-size, breakpoints and site--grid are set in EM
- open browser settings and increase default font-size.
- see result

If all sizes are set in relative sizes the website grows equally when default font-size is enlarged or reduced
